src/test/java/seedu/address/model/person/PriorityTest.java

```diff
@@ -0,0 +1,119 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class PriorityTest {
+    private static final Priority PRIORITY_HIGH = new Priority("high");
+    private static final Priority PRIORITY_MEDIUM = new Priority("medium");
+    private static final Priority PRIORITY_LOW = new Priority("low");
```

Comment:
is this necessary? You can just reuse the constant u declared in `Priority` class

Reply:
Changed